feature/feature.go

```diff
@@ -108,4 +116,6 @@ var defaultCAPAFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	AlternativeGCStrategy:         {Default: false, PreRelease: featuregate.Alpha},
 	TagUnmanagedNetworkResources:  {Default: true, PreRelease: featuregate.Alpha},
 	ROSA:                          {Default: false, PreRelease: featuregate.Alpha},
+	AWSMachine:                    {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
I believe both feature gates should be beta not alpha

Reply:
Fixed